contrib/fsl_integration/sct_plugin.py

```diff
@@ -3,13 +3,22 @@
 #########################################################################################
 # This code will provide SCT integration into fsleyes for the following tools:
 #
+#    - sct_propseg
 #    - sct_deepseg_gm
 #    - sct_deepseg_sc
+#    - sct_label_vertebrae
+#    - sct_register_to_template
+#    - sct_process_segmentation
+#    - sct_dmri_moco
+#    - sct_dmri_compute_dti
+#
 #
 # ---------------------------------------------------------------------------------------
 # Copyright (c) 2018 Polytechnique Montreal <www.neuro.polymtl.ca>
 # Authors: Christian S. Perone
+#          Thiago JR Rezende
 # Created: 2 Mar 2018
+# Uptaded: 11 Jul 2019
```

Comment:
no need for this field, github takes care of the versionning

Reply:
Just for record, all functions that does not work inside GUI, works outside (running by the terminal, I mean).